### PR TITLE
support only drf 3.12

### LINKED ISSUE #832: Add DRF 3.12 support
See https://www.django-rest-framework.org/community/3.12-announcement/

### PR DESCRIPTION
**Pull request by n2ygk**
Fixes #832 

## Checklist

- [x] PR only contains one change (considered splitting up PR)
- [ ] unit-test added
- [x] documentation updated
- [x] `CHANGELOG.md` updated (only for user relevant changes)
- [x] author name in `AUTHORS`

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,16 +1,15 @@
 [tox]
 envlist =
     py{35,36,37}-django22-drf{310,311,master},
```
Comment (sliverc) [inline tox.ini:3]: it should be possible to combine this line and the next two into one with something like the following (not tested):

```
py{35,36,37,38}-django{22,30}-drf{312,master}
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,11 @@ any parts of the framework not mentioned in the documentation should generally b
 
 * Removed support for Django 1.11.
 * Removed support for Django 2.1.
+* Removed support for Django REST framework 3.10, 3.11
+
+### Added
+* Added support for Django REST framework 3.12
```
Comment (sliverc) [inline CHANGELOG.md:20]: in release 3.2.0 below please also note, that that will be the last release supporting DRF 3.10 and DRF 3.11

**Review by sliverc — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (auvipy) [reply tox.ini:3]: python 3.5 is EOL

Comment (n2ygk) [reply tox.ini:3]: > python 3.5 is EOL

Django 2.2 supports python 3.5.

Comment (n2ygk) [reply tox.ini:3]: > it should be possible to combine this line and the next two into one with something like the following (not tested):
> 
> ```
> py{35,36,37,38}-django{22,30}-drf{312,master}
> ```

I believe py35 is not supported by django 3, hence the separation there but I can combine a couple of other lines. Testing now.

Comment (sliverc) [reply tox.ini:3]: fair point Django 3 indeed only supports 3.6+

**Comment by n2ygk** [comment]
Wait. this latest push is incorrect. Should only test drf 3.12... Standby, testing again. This is not meant to be merged until 4.0....

**Review by sliverc — APPROVED** [review]

**Comment by sliverc** [comment]
We already have backwards incompatible changes in master so next version will be 4.0.0 in any case. So merging this.

Thanks.

**Pull request merged**